mkt/websites/indexers.py

```diff
@@ -32,29 +33,32 @@ def get_mapping(cls):
                     'description': {'type': 'string',
                                     'analyzer': 'default_icu'},
```

Comment:
In webapps, we only have `'position_offset_gap': 100` for `name`. I'll steal it for `title`, should I add it for `description` in both webapps and websites ?